config/install/field.storage.media.field_media_oembed_video.yml

```diff
@@ -3,13 +3,15 @@ status: true
 dependencies:
   module:
     - media
-    - text
-id: media.field_media_image_caption
```

Comment:
No more captions?

Reply:
Captions are handled directly in the WYSIWYG. And can be implemented at the component level. As earth has described, most captions are contextual to the surrounding content.

Reply:
I'm re-opening [D8CORE-106](https://stanfordits.atlassian.net/browse/D8CORE-106)